org.jacoco.core/src/org/jacoco/core/internal/Java9Support.java

```diff
@@ -41,9 +41,6 @@ private Java9Support() {
 	 */
 	public static byte[] readFully(final InputStream is)
 			throws IOException {
-		if (is == null) {
```

Comment:
@marchof I guess that you misread change: it doesn't add check, but removes it exactly because we don't need it - indeed in all places for a `null` `InputStream` we throw `NullPointerException` instead of `IllegalArgumentException`, also this condition wasn't covered by tests, so I took opportunity to remove it in this PR.